google_compute_engine/instance_setup/instance_config.py

```diff
@@ -41,7 +41,7 @@ class InstanceConfig(config_manager.ConfigManager):
   instance_config_options = {
       'Accounts': {
           'deprovision_remove': 'false',
-          'groups': 'adm,dip,lxd,plugdev,video',
+          'groups': 'adm,dip,lxd,plugdev,video,docker',
```

Comment:
Also, can you make this PR against the development branch? We don't check in directly to master.